### DISCUSSION TITLE
Batch counter issue with 2.4.6

### DISCUSSION THREAD

**Discussion by stuartmcknight** [artisan-roaster-scope/artisan#669] [Q&A]
Anybody else having problems with Batch counter ?

Enabled Batch Counter in configuration. Say set to 253 as initial count.

When creating a new roast, from the title bar it appears that the batch number is correctly displayed using ~batch as 253 with the rest of the roast properties,

However, once roast completes, the Autosave has now changed the 253 to 232. And when reopening the Batch dialog, the new seed number seems to have DECREMENTED to 231. Artisan has not been restarted during these runs so I don't believe it has reloaded the config (although the don't overwrite checkbox is enabled anyway)

Searched here but came up empty so either I'm lucky, nobody else is using batch counter or aliens have taken over my machine....

**Comment by CarefreeBuzzBuzz** [comment]
Maybe you never hit Drop to confirm the new number.  

https://artisan-scope.org/docs/batches/

Michael Herbert 
CarefreeBuzzBuzz

**Comment by stuartmcknight** [comment]
I did. But that doesn’t explain why it decremented from 253 to 252 then 251…

--Stuart

**Comment by roasterdave** [comment]
Please make any additional comments in the previously filed Issue #667.  Thanks.
